### DISCUSSION TITLE
Found a "broken" bookmark, how to debug what went wrong?

### DISCUSSION THREAD

**Discussion by ovizii** [karakeep-app/karakeep#642] [Q&A] [answered]
This is what I noticed:

Screenshot 1:
![image](https://github.com/user-attachments/assets/3ab8e75e-e11a-4b51-87d7-50cb404096da)

Screenshot 2:
![image](https://github.com/user-attachments/assets/be639c2a-6b81-4e76-acd9-08246aad069f)

Screenshot 3:
![image](https://github.com/user-attachments/assets/7dfe645c-0f57-4a90-85eb-3cf449fd22ba)

- Is this a crawling error?
- Is there a way to trigger re-crawling / indexing / inferencing for a specific bookmark maybe using the 3 dots seen in screenshot 2?

**Comment by kamtschatka** [comment]
Looks like they are using Cloudflare to make sure you do not scrape the page. Nothing really you can do about this. The whole point of this feature is to keep you from scraping, so we are not going to start a fight to avoid being detected.

You can trigger a refresh of the bookmark by clicking the "Refresh" button in the "..." menu.

**Reply by ovizii** [reply]
Thanks for confirming that this is indeed a crawling error.

I am not sure about the other part of your reply though. 
- It sounds like “Refresh” triggers a re-crawl, right?

Also:
- Should I rather delete the bookmark and add it again, this time via the browser extension? (I read somewhere that does allow me to manually complete the CF challenge)
- Is this the “1! Crawling error shown in my screenshot 3?
- If it is, is there any way to find this “1” crawling error or if this happens again, would I need to manually go through all my bookmarks to find the culprit?

**Reply by kamtschatka** [reply, marked as answer]
yes, refresh triggers a recrawl. 

* the browser extension does not help here. I am not sure where you read that it allows you to complete the CF challenge. There is an issue to implement it that way though.
* I don't know if that is the "1" showing in the screenshot, this is currently only available in the db as far as I know. There are plans however to add a search in the future, which would be able to filter on those kinds of properties.

**Reply by ovizii** [reply]
You are right, I remembered wrong, this is what I had read but you are right, it is still a feature request: https://github.com/hoarder-app/hoarder/issues/172
